app/services/performance.ts

```diff
@@ -5,7 +5,7 @@ import { StatefulService, mutation } from './core/stateful-service';
 import electron from 'electron';
 import * as obs from '../../obs-api';
 
-interface IPerformanceState {
+export interface IPerformanceState {
```

Comment:
Please create a copy of this interface in the external-api/performance/performance.ts

The interface in the internal API may be changed in the future and it should not affect the external API. So external API should not expose interfaces from the internal API.

Reply:
Makes sense, thanks!